### PR TITLE
Horizon 2.0

### PR DESCRIPTION
**Pull request by chipotle**
These changes are all addressing #104 (and incorporate #99).

### CODE REVIEW

File: authentication.md [hunk 1]
```diff
@@ -63,7 +64,7 @@ In order to use OAuth with Horizon, you'll need to configure a TLS certificate s
 
 [cf]: /docs/configuration
 
-You'll need to enter `client_id` and `client_secret` values in the `.hz/config.toml` file for your server. Toward the bottom of the automatically generated file, you'll see commented-out sample OAuth settings. Uncomment the appropriate lines and replace the dummy values with the ones you've received from the application provider. Adding Github OAuth configuration data would look like this:
+You'll need to enter `id` and `secret` values in the `.hz/config.toml` file for your server (along with `host` and `redirect_url` values for Auth0). Toward the bottom of the automatically generated file, you'll see commented-out sample settings. Uncomment the appropriate lines and replace the dummy values with the ones you've received from the application provider. Adding Github OAuth configuration data would look like this:
```
Comment (deontologician) [inline authentication.md]: The id and secret should go into the `.hz/secrets.toml` file

File: getting-started.md [hunk 2]
```diff
@@ -45,6 +47,10 @@ Here's what these files and directories are:
 * `src` is for source files for your build system. This isn't a convention you have to follow; Horizon doesn't touch anything in this directory.
 * `dist/index.html` is a sample file. You'll replace this as you develop your application, but there's enough in it to verify that Horizon is installed and working.
 * `.hz/config.toml` is a [TOML][] configuration file for the Horizon server.
+* `.hz/schema.toml` is the default schema file. Initially, it's blank.
```
Comment (deontologician) [inline getting-started.md]: Initially `schema.toml` has the default schema which is an admin group and a rule that the admin group has carte blanche to execute any query

File: permissions.md [hunk 3]
```diff
@@ -110,22 +110,28 @@ validator = """
...
+You can extract the current schema from a Horizon cluster with `hz schema save`:
 
 ```bash
-# Export the current schema into `current_schema.toml`
-$ hz get-schema -o current_schema.toml
+# Save the current schema to current_schema.toml
+$ hz schema save -o current_schema.toml
+
+# Save the current schema to the default file (.hz/schema.toml)
+$ hz schema save
```
Comment (dalanmiller) [inline permissions.md]: I'd say put this example first, if someone is quickly scanning how to save the current schema, I'd like them to hit this one first. This makes debugging easier if we encourage the use of `schema.toml`

**Comment by dalanmiller** [comment]
LGTM @chipotle 👍

**Comment by chipotle** [comment]
Pinging for review, although @dalanmiller may not be around to review it. :)

File: limitations.md [hunk 4]
```diff
@@ -23,8 +23,8 @@ Setting up a Horizon server under Windows has not been extensively tested.
 
 ## Limitations
 
+* Collection names may not start with  `hz_`. Using this prefix will result in an error.
```
Comment (deontologician) [inline limitations.md]: I think this should move off of the limitations page and go to the collections page as a note. This isn't a limitation like the others where we expect to fix it, it's just how horizon works now

File: cli.md [hunk 5]
```diff
@@ -32,14 +34,25 @@ Every Horizon server requires a RethinkDB server to connect to. Use the `--conne
...
 * `--cert-file PATH` The certificate to use for the HTTPS server. Default: `./horizon-cert.pem`
+* `--rdb_timeout SECONDS`: timeout to make the connection to the RethinkDB cluster, in seconds. Default: 20
+
+[rdbopts]: /docs/configuration/#rdbopts
+
+The following options provide alternatives to specifying these values in the `--connect` string:
+
+* `--rdb_host HOSTNAME`
+* `--rdb_port PORT`
+* `--rdb_user USERNAME`
```
Comment (danielmewes) [inline cli.md]: I think the `rdb_user` and `rdb_password` options also need to be added to embed.md .

**Comment by deontologician** [comment]
Got another one:
We need to change the example auth code snippet from saying:
```js
window.location.pathname = endpoint;
```
to
```js
window.location.replace(endpoint);
```
The endpoints are now fully qualified urls, so setting pathname will result in stuff like `http://localhost:8181/http://localhost:8181/horizon/github` etc.

**Pull request merged**